scripts/linters/pylint_extensions_test.py

```diff
@@ -3311,7 +3325,8 @@ def __init__(self, fake_arg):
         with checker_test_object.assertAddsMessages(
             testutils.Message(
                 msg_id='no-dunder-metaclass',
```

Comment:
I'm a bit confused by this, it seems specific to our codebase -- what does "dunder" mean?

Reply:
I'm not sure, but it comes from `DisallowDunderMetaclassChecker`.

Reply:
Could you please check with @Hudda, @DubeySandeep or @AdityaDubey0?

Reply:
Pinged them.

Reply:
Dunder means double underscore* no-dunder-meta class helps use disallow usage of ‘__metaclass__’ instead asks developers to use meta less defined in python utilise.

Reply:
Ah ok, makes sense, thanks!